tooling/tslint-rules/awaitReturnedPromiseRule.ts

```diff
@@ -0,0 +1,48 @@
+import * as ts from "typescript";
+import * as tslint from "tslint";
+import { isIdentifier } from 'tsutils';
+
+const AWAIT_RETURNED_PROMISE = `must explicitly await returned promise (see https://github.com/FlowCrypt/flowcrypt-browser/pull/2349)`;
+
+export class Rule extends tslint.Rules.TypedRule {
```

Comment:
I tried to write this as ESLINT rule but failed. So it will have to be a tslint rule, at least for now.